### PR TITLE
[hive/app] add query builder for laboratory

### PR DESCRIPTION
**Pull request by dimaMachina**
all fixed issues listed here https://github.com/graphql/graphiql/pull/3633

before:

https://github.com/kamilkisiela/graphql-hive/assets/7361780/b0feb35b-d8ce-454f-923a-18765fa7f15e

after:

https://github.com/kamilkisiela/graphql-hive/assets/7361780/fa54628e-abf3-4590-b2e2-c014f097563f

### CODE REVIEW

**Comment by kamilkisiela** [comment]
Screenshot 2024-06-26 at 12 33 43
that's one giant dollar sign :D

**Comment by dimaMachina** [comment]
> Screenshot 2024-06-26 at 12 33 43 that's one giant dollar sign :D



> Screenshot 2024-06-26 at 12 33 43 that's one giant dollar sign :D

lol, yes, it's some bug from this plugin 😂

File: packages/web/app/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@fastify/cors": "9.0.1",
     "@fastify/static": "7.0.4",
     "@fastify/vite": "6.0.6",
+    "@graphiql/plugin-explorer": "3.1.0-canary-1fa82fbf.0",
```
Comment (dotansimha) [inline packages/web/app/package.json:20]: can we use stable?

**Review by dotansimha — APPROVED** [review]

File: packages/web/app/package.json (hunk 1 above)
Comment (dimaMachina) [reply packages/web/app/package.json]: updated

**Review by kamilkisiela — CHANGES_REQUESTED** [review]
https://github.com/kamilkisiela/graphql-hive/assets/8167190/e45e3e1b-63af-418b-ae78-fbcc65af7965


This element is super weird. It's small, only available when you hover on the field and hard to click as it's tiny and you know what it does only when you leave your mouse cursor on it for 7s.

https://github.com/kamilkisiela/graphql-hive/assets/8167190/88515b20-9fff-4eb0-8fc5-8b8d4319818b

And borders / backgrounds.

Screenshot 2024-07-10 at 11 40 42

**Comment by dimaMachina** [comment]
> This element is super weird. It's small, only available when you hover on the field and hard to click as it's tiny and you know what it does only when you leave your mouse cursor on it for 7s.

fixed in plugin

> And borders / backgrounds.

fixed in plugin

**Comment by n1ru4l** [comment]
There should be more visual separation between fields and field arguments
image

Not a blocker for me though.

**Review by kamilkisiela — APPROVED** [review]

**Pull request merged**